### PR TITLE
Sponsor orbit

### LINKED ISSUE #70: Add Sponsor model using yaml
I would like to maintain the sponsors as models using YAML files. Similar to the FAQ model.

### PR DESCRIPTION
**Pull request by codedge**
Closes #70 

This PR stores the sponsors in `yml` files.

### CODE REVIEW

**Review by Gummibeer — APPROVED** [review]
Great work! 🎉🚀
Only the priority isn't needed - I would like to go for alphabetical order. The current template was like `created_at` ordered but I don't want to judge who comes first - so alphabetical is an objective way.

You can do it - otherwise I will do it tomorrow right after merge. ☺️

**Comment by codedge** [comment]
Doing it now 😉

File: app/Models/Sponsor.php [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+use Illuminate\Database\Schema\Blueprint;
+use Orbit\Concerns\Orbital;
+use Spatie\EloquentSortable\Sortable;
+use Spatie\EloquentSortable\SortableTrait;
+use Spatie\Sluggable\HasSlug;
+use Spatie\Sluggable\SlugOptions;
+
+class Sponsor extends Model implements Sortable
+{
+    use SortableTrait;
```
Comment (Gummibeer) [inline app/Models/Sponsor.php:15]: That trait and all it's related stuff can also be removed.

Comment (codedge) [reply app/Models/Sponsor.php]: Fixed

**Pull request merged**